### DISCUSSION TITLE
Gameplay Direction

### DISCUSSION THREAD

**Discussion by benjamin-kirkbride** [pythonarcade/community-rpg#31] [Ideas]
We are in the extremely early stages of the game, and everything is still in flux. This thread will track discussion of gameplay.

The important thing to keep in mind is that the purpose of this project is to enable people to get started quickly, so any gameplay decisions must keep in mind how complex it would be to implement, and how difficult it will make on-boarding for future new and inexperienced contributors.

**Comment by iansedano** [comment]
Two of the overarching decisions are:

- _Goal of game_
  - **Boss based with final boss**: preferred because it opens the door for more contributions
  - High Score based
- _Fighting Style_
  - **Turn based**: preferred because easier
  - Real time

Below is a list of elements that could make up either style:

- Items
  - Tools
  - Weapons
  - Power-ups
  - Consumables
  - Materials
- Main Character
  - Inventory
  - Health
  - Hunger
  - XP
  - Skills
- Non Player Characters (NPCs)
  - Animals
  - Friends
  - Enemies
  - Bosses
- Crafting?

**Reply by pushfoo** [reply]
Shops and shop keepers should be considered as well.

**Comment by pushfoo** [comment]
I have a simple and actionable idea: 
Make the torch and other light sources into acquirable items.

Questions I have remaining:
1. Should it be mutually exclusive with other hot bar items?
2. Should it be time-limited?
